cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go

```diff
@@ -54,7 +56,7 @@ func runControlPlane() func(c workflow.RunData) error {
 
 		// if this is not a control-plane node, this phase should not be executed
 		if !data.IsControlPlaneNode() {
-			fmt.Println("[upgrade] Skipping phase. Not a control plane node.")
+			fmt.Println("[upgrade/control-plane] Skipping phase. Not a control plane node.")
```

Comment:
i see you are adding `upgrade/foo` as part of this PR. i think in the previous PR we removed the `/foo` part from multiple places. i don't think we are consistent about this format in `init` and `join` too, but leaving the decisions to you since i haven't see how this PR runs after the changes. i.e. maybe the `/foo` part makes things more clear.

Reply:
i think our current state is a bit messy, but seems like something we can refactor in the future.